.gitignore

```diff
@@ -17,12 +17,10 @@ torch_xla/csrc/generated/
 #
 # Below files are not deleted by "setup.py clean".
 
-third_party/tensorflow/
```

Comment:
I think we currently still have patch for `tf`, would those show up if we remove `tf` from ignore list?

Reply:
Nope, just tested this. In both cases, `git status` for pytorch/xla doesn't show the TF changes and `git status` in the TF submodule does.